src/Config/Settings.ts

```diff
@@ -198,3 +201,15 @@ export const toggleFeature = (internalName: string, enabled: boolean): FeatureFl
 export function getUserSelectedTaskFormat(): TaskFormat {
     return TASK_FORMATS[getSettings().taskFormat];
 }
+
+/**
+ * Retrieves the global {@link Query}
+ *
+ * @exports
+ * @returns {Query}
+ */
+export function getGlobalQuery(): Query {
+    const { globalQuery } = getSettings();
+
+    return new Query({ source: globalQuery });
+}
```

Comment:
Ahhh - this use of Query is the source of the extra cyclic dependencies mentioned in:
https://github.com/obsidian-tasks-group/obsidian-tasks/pull/1878#issuecomment-1520771558

I do really like that you've hidden access to `getSettings()` inside a function... 

Because Query depends on so much, I suggest that the return type of this function is changed to string, so that the `Query` is constructed by the caller of `getGlobalQuery()`.